macros/src/main/scala/enumeratum/compat/EnumMacros.scala

```diff
@@ -10,7 +11,7 @@ object EnumMacros {
 
   /** Finds any [A] in the current scope and returns an expression for a list of them
     */
-  def findValuesImpl[A: c.WeakTypeTag](c: Context): c.Expr[IndexedSeq[A]] = {
+  def findValuesImpl[A: c.WeakTypeTag](c: Context): c.Tree = {
```

Comment:
Was this needed? Feels like a step backwards to lose type info, so if it's not required, please revert.